### ISSUE TITLE
Make external links box on viewgame page more screen reader friendly

### ISSUE THREAD

**Issue by alice-blue** [iftechfoundation/ifdb#1389]
The external links box has file type icons which are links and, beside them, text links that lead to the same destination.

For at least one screen reader user, the screen reader announces a file name when it gets to the icon link, presumably because no alt text is provided for the icons.

I got a suggestion to handle it this way:

* Hide the icon links using aria-hidden and tabindex. (The story file type is usually mentioned in adjacent text anyway.) 
* Add alt-text to the graphic links as a fallback in case ARIA isn't honored. For story files, it was suggested that the alt text be "Download" followed by the same text used in the text link. For non-story files, just use the same text that's in the text link.

Related to https://github.com/iftechfoundation/ifdb/issues/1388 and https://github.com/iftechfoundation/ifdb/issues/1377
